lib/src/web_socket/browser/sockjs.dart

```diff
@@ -37,7 +38,8 @@ class SockJSSocket extends CommonWSocket implements WSocket {
     sockjs.Client client = new sockjs.Client(uri.toString(),
         debug: debug == true,
         noCredentials: noCredentials == true,
-        protocolsWhitelist: protocolsWhitelist);
+        protocolsWhitelist: protocolsWhitelist,
+        timeout: timeout.inMilliseconds);
```

Comment:
tests failing - looks like something related to a null timeout value here so it looks like you need some fallback here